course2/public/guess.js

```diff
@@ -0,0 +1,206 @@
+// guess the word module
+let guessTheWord = function () {
+	// "config" options
+	const MAX_FAILS = 6;
+	const BONUS_PER_LETTER = 5; // bitcoins
+	const chosenWord = [];
```

Comment:
Since this is a variable that will change, it makes sense to declare it with `let` instead of `const`.

Reply:
Done!